core/src/scss/components/link/_link--internal.scss

```diff
@@ -7,5 +7,12 @@
 // @see scss/utilities/mixins/link/_link-icon.scss
 //
 .su-link--internal {
-  @include link-icon(lock, 0.75em);
+  @include link-icon(lock, 0.75em, none, -0.06em);
+
+  &:hover,
+  &:focus {
+    &::after {
+      mask-image: url("#{$su-image-path}/lock-solid.svg");
```

Comment:
Support looks good for this and the autoprefixer rendered the right prefixes.
https://caniuse.com/#search=mask-image